clients/docs/ccloud.rst

```diff
@@ -0,0 +1,236 @@
+.. _client-examples-ccloud:
+
+Confluent Cloud CLI
+-------------------
+
+In this tutorial, you will run a Confluent Cloud CLI client application that
+produces messages to and consumes messages from an |ak-tm| cluster.
+
```

Comment:
@ybyzek Is there a replacement token for "Confluent Cloud CLI?"

Reply:
@tikimims 

Confluent Cloud CLI.--> `|ccloud| CLI`
Confluent CLI --> `|confluent-cli|`

But for

Reply:
Great! Thank you. Updated